src/cache.js

```diff
@@ -1,17 +1,26 @@
 import DataLoader from 'dataloader'
 
+const remapDocs = (docs, ids) => {
+  const idMap = {}
+  docs.forEach(doc => {
+    idMap[doc._id] = doc
+  })
+  return ids.map(id => idMap[id])
+}
```

Comment:
Is this really necessary ?

Reply:
Looks like it: https://github.com/graphql/dataloader#batch-function